doc/guide.md

```diff
@@ -3142,9 +3138,19 @@ during the operation.
 
 Add 'loadSample=true` to parameters, to load sample data as well.
 
+If the module supports version 2 of the `_tenant` interface it should return status 201 with
+a `Location` header. This signals that the tenant job has started. Use a GET request on the
+Location path returned to poll for the completion of the tenant job with the
+
+    curl -HX-Okapi-url:http://localhost:8081 -HX-Okapi-Tenant:testlib \
+      -HContent-Type:application/json "-HAccept:*/*" \
```

Comment:
```suggestion
    curl -H "X-Okapi-url: http://localhost:8081" -H "X-Okapi-Tenant: testlib" \
      -H "Content-Type: application/json" -H "Accept: */*" \
```

Reply:
More typing, but I guess it's more readable.